Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/BaseClasses/Effectiveness.mo

```diff
@@ -1,108 +1,99 @@
 within Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.BaseClasses;
 model Effectiveness
-  "Model for calculating the heat exchange effectiveness of heat exchangers"
+  "Model for calculating the heat exchange effectiveness"
   extends Modelica.Blocks.Icons.Block;
-
-  parameter Modelica.Units.SI.Efficiency epsS_cool_nominal(max=1) = 0.8
+  parameter Modelica.Units.SI.Efficiency epsSenCoo_nominal(final max=1) = 0.8
     "Nominal sensible heat exchanger effectiveness at the cooling mode";
-  parameter Modelica.Units.SI.Efficiency epsL_cool_nominal(max=1) = 0.8
+  parameter Modelica.Units.SI.Efficiency epsLatCoo_nominal(final max=1) = 0.8
     "Nominal latent heat exchanger effectiveness at the cooling mode";
-  parameter Modelica.Units.SI.Efficiency epsS_cool_partload(max=1) = 0.75
+  parameter Modelica.Units.SI.Efficiency epsSenCoo_ParLoa(final max=1) = 0.75
```

Comment:
We may change the parameter name to `epsSenCooPL`, see for example the variable `etaPL` in `Buildings.Fluid.Chillers.BaseClasses.Carnot`. And also change in the comment the `Partial load` to `Part load`.

Please apply the pattern to other similar parameters.